llvm/test/CodeGen/AArch64/sme2-mops.ll

```diff
@@ -0,0 +1,66 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+; RUN: llc -mtriple=aarch64-linux-gnu -mattr=+sme2 -verify-machineinstrs < %s | FileCheck %s
```

Comment:
I think we should have a way to disable this behaviour, in case there are any issues with this approach. Can you add an option for this, and change this test to have two RUN lines (one with it enabled, one with it disabled)?

Reply:
Done.